scripts/install.sh

```diff
@@ -528,6 +528,23 @@ elif [ "$format" = "tar" ]; then
       fi
     fi
 
+    # check for an existing Doppler binary
+    existing_install_dir=$(which doppler)
+    if [ "$existing_install_dir" != "" ]; then
+      install_dir="$existing_install_dir"
+      # capture exit code without exiting
+      set +e
+      install_binary "$install_dir"
+      exit_code=$?
+      set -e
+      if [ $exit_code -eq 0 ]; then
+        binary_installed=1
+        BINARY_INSTALLED_PATH="$install_dir"
+      elif [ $exit_code -eq 1 ]; then
+        found_non_writable_path=1
+      fi
```

Comment:
Currently, if the CLI already exists but we're not able to overwrite it, we'll keep executing and ultimately install a second copy in one of the default paths. This is probably ok, but wanted to call it out.

Reply:
Per our discussion we're going to leave this as-is. This situation shouldn't come up too frequently and if it does, it's not a huge issue. If it comes up as a problem later, we can revisit it.